main.sh

```diff
@@ -1,5 +1,3 @@
 #!/usr/bin/env bash
-
 export $(egrep -v '^#' .env | xargs)
-
```

Comment:
Is there a reason for these whitespace changes?

Reply:
no, not really. must have done that inadvertantly.

note the env command was just removed in the latest push.

Reply:
Why remove the env command? It's handy for deploying outside of Docker.

Reply:
i think if you want to keep the .env file we will have to put it back in the container so the main.sh launch doesnt complain.

i specify environment variables within the IDE for instances such as these. and on the deploy side env params can be specified in the helm charts.